app/models/asset_field_type.rb

```diff
@@ -52,6 +52,15 @@ def mapping
 
   private
 
+  def is_image?(mime_type)
+    ['image/jpeg', 'image/pjpeg', 'image/png','application/pdf' ,'image/x-png', 'image/gif'].include?(mime_type)
```

Comment:
Few things: are you replying the right comment? Is that the correct method you pasted? I'm not sure I understand what you're saying